hpat/_distributed.cpp

```diff
@@ -455,6 +461,16 @@ int hpat_finalize()
     return 0;
 }
 
+void permutation_int(int64_t* output, int n)
+{
+     if (hpat_dist_get_rank() == 0) {
```

Comment:
There is no need to generate permutation in C because `output` already has a permutation array from Python.